data/_models/shared/Studies.csv

```diff
@@ -0,0 +1,26 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
```

Comment:
studies also exist in datacatalogue, but as a subclass of Resources.

(offtopic: in the yaml we need to decide if we 
- put all subclasses in one file (then we don't need to set 'order' on columns)
- have seperate file per subclass (then we must explicitly set 'order' in columns when columns should not be addede at bottom

I have been thinking that we need another composition mechanism, where we put 'sections' of a table in seperate file and then being able to 'include' those. That combined with 'all subclasses in one file' might be very powerful)

Reply:
Indeed such a 'reuse' mechanism might be extremely nice, I see the same for many complex Beacon datatypes but also simpler things like datamodels where every class has an id, name, title, description and timestamp (or something).